pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -2554,7 +2571,7 @@ func GetStatus[I, IS any](spec I) IS {
 		return any(t.Status).(IS)
 	case *gatewayx.XBackendTrafficPolicy:
 		return any(t.Status).(IS)
-	case *gatewayalpha3.BackendTLSPolicy:
+	case *k8s.BackendTLSPolicy:
```

Comment:
So does this mean we've officially dropped support for v1alpha BackendTLS policies?